### PR TITLE
Add information for CSS usage with CSS variables

### PR DESCRIPTION
**Pull request by alex-page**
### WHY are these changes introduced?

Addresses comment in https://github.com/Shopify/polaris-styleguide/pull/3679#issuecomment-767914590

### WHAT is this pull request doing?

Adds some documentation

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
- [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

**Comment by kaelig** [comment]
Adding this to the body (or somewhere else?) might also be needed?

```
style="background-color: rgb(246, 246, 247); color: rgb(32, 34, 35);"
```

And for dark mode (useful to know for internal Shopify teams, but not sure it should be advertised to 3rd party devs):

```
style="background-color: rgb(11, 12, 13); color: rgb(227, 229, 231)"
```

File: src/components/README.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ ReactDOM.render(app, domContainerNode);
 
 ### CSS components
 
+Add the [latest CSS variables](https://unpkg.com/browse/@shopify/polaris-tokens@latest/dist-modern/palette/base.light.polaris.custom-properties.css) from the polaris tokens library.
```
Comment (kaelig) [inline src/components/README.md:98]: What does this mean from an implementation perspective??

I was going to go for this approach in my project, but perhaps there's another recommended way?

```html
    <link
      rel="stylesheet"
      href="https://unpkg.com/@shopify/polaris-tokens@2.18.0/dist-modern/palette/base.dark.polaris.custom-properties.css"
    />
    <link
      rel="stylesheet"
      href="https://unpkg.com/@shopify/polaris@6.0.0-alpha.7/dist/styles.css"
    />
```

**Comment by kaelig** [comment]
Oh, and I just found out that some tokens were missing from this file https://unpkg.com/@shopify/polaris-tokens@2.18.0/dist-modern/palette/base.dark.polaris.custom-properties.css, such as `--p-border-radius-base`. Looking into it.

**Comment by alex-page** [comment]
`--p-border-radius-base` is going to be deprecated. Maybe that should happen this release.

**Comment by kaelig** [comment]
Thanks! I'll add the tokens coming from https://github.com/Shopify/polaris-react/blob/6e4d0682c03066674b3e7468f09201014d27e438/src/utilities/theme/tokens.ts directly into my project in the meantime, and will delete them when they're not needed anymore!

**Review by BPScott — APPROVED** [review]

**Comment by kaelig** [comment]
Thanks for being so reactive!

**Pull request merged**